Sources/FoundationInternationalization/TimeZone/TimeZone_ICU.swift

```diff
@@ -30,7 +30,8 @@ internal final class _TimeZoneICU: _TimeZoneProtocol, Sendable {
         fatalError("Unexpected init")
     }
     
-    struct State {
+    // This type is safely sendable because it is guarded by a lock in _TimeZoneICU and we never vend it outside of the lock so it can only ever be accessed from within the lock
+    struct State : @unchecked Sendable {
```

Comment:
This resolves a warning about `lock` not being concurrency safe below with an explanation on why this is safe